app/services/claims/payments/claim/generate_csv_file.rb

```diff
@@ -28,6 +29,7 @@ def call
           claim.school.urn,
           claim.school_name,
           claim.school.local_authority_name,
+          claim.provider.name,
```

Comment:
```suggestion
          claim.provider_name,
```

Can we use delegation here?